src/python/test/test_cubical_complex.py

```diff
@@ -158,6 +189,26 @@ def test_cubical_generators():
     assert np.array_equal(g[0][1], np.array([[7, 4]]))
     assert np.array_equal(g[1][0], np.array([8]))
 
+def test_cubical_generators_from_vertices():
+    cub = CubicalComplex(vertices = [[0, 0, 0, 0], [0, 1, 1, 0], [0, 1, 1, 0], [0, 0, 0, 0]])
```

Comment:
Again a square matrix :cry: I need to replace it with a rectangle (or add another test with a rectangle) to let the test detect more possible mistakes.